README.md

```diff
@@ -4,51 +4,49 @@
 
 # Run inference at scale
 
-Cortex is an open source platform for large-scale inference workloads.
+Cortex is an open source platform for large-scale machine learning inference workloads.
 
 <br>
 
-## Model serving infrastructure
+## Architecture
 
-* Supports deploying TensorFlow, PyTorch, and other models as realtime or batch APIs.
-* Ensures high availability with availability zones and automated instance restarts.
-* Runs inference on on-demand instances or spot instances with on-demand backups.
-* Autoscales to handle production workloads with support for overprovisioning.
+* Kubernetes - container orchestration
+* Istio - traffic management
+* Prometheus - metric aggregation
+* Fluent Bit - structured logging
+* FastAPI - request handling
+* TensorFlow Serving / ONNX Runtime - model serving
 
-#### Configure a cluster
-
-```yaml
-# cluster.yaml
+<br>
 
-region: us-east-1
-instance_type: g4dn.xlarge
-min_instances: 10
-max_instances: 100
-spot: true
-```
+## Workloads
 
-#### Spin up on your AWS or GCP account
+### Realtime APIs - respond to prediction requests on demand
```

Comment:
I'm not sure if "on demand" is the best phrasing. Maybe "synchronously"? Any other ideas?